### PR TITLE
Add KIDDO Token

### PR DESCRIPTION
**Pull request by divyesh-puri**
Description
Added KIDDO L1 and L2 token details to the tokens list

Additional context
Challenge to bridge KIDDO token to optimism

**Metadata**
- Fixes #[Link to Issue]

### CODE REVIEW

**Comment by divyesh-puri** [comment]
@elenadimitrova Sorry for tagging but don't know how else to get someone to review the PR.

**Comment by divyesh-puri** [comment]
@gigamesh Sorry for tagging but its been two days and no one has reviewed or merged this.

**Comment by gigamesh** [comment]
> @gigamesh Sorry for taggi

No problem, but I'm no longer at Optimism so I can't help here. 

Hi @elenadimitrova ! :D

**Comment by elenadimitrova** [comment]
I am no longer working for Optimism as well. Maybe check in Discord who's best to talk to?

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -1371,6 +1371,28 @@
...
+        "optimismBridgeAddress": "0x0b651A42F32069d62d5ECf4f2a7e5Bd3E9438746"
+      }
+    },
+    {
+      "chainId": 69,
+      "address": "0x0E150963D01e8453B7099a3C032345527A4BAfFD",
+      "name": "KIDDO",
+      "symbol": "KIDDO",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BitBTC.png",
```
Comment (Nicca42) [inline optimism.tokenlist.json:1392]: Hi @divyesh-puri! Your PR is using the BitBTC logo. Can you confirm this is correct? If so I will need to get confirmation from the BitBTC team that this is OK.

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: optimism.tokenlist.json (hunk 1 above)
Comment (divyesh-puri) [reply optimism.tokenlist.json]: Whoops missed that!! @Nicca42 Fixed it.

**Comment by divyesh-puri** [comment]
@Nicca42 Can you take a look here once. Please

**Review by Nicca42 — CHANGES_REQUESTED** [review]
Please rebase master or CI will not be able to run on your PR.

File: optimism.tokenlist.json [hunk 2]
```diff
@@ -1367,9 +1367,25 @@
       "name": "Equalizer",
       "symbol": "EQZ",
       "decimals": 18,
-      "logoURI": "https://ethereum-optimism.github.io/logos/EQZ.png",
+      "logoURI": "https://ethereum-optimism.github.io/logos/EQZ.png"
```
Comment (Nicca42) [inline optimism.tokenlist.json:1370]: You have broken the EQZ token listing here by removing the bridge extension. As such CI will never pass. 

Please fix this by adding back the below code above your token listing. 

> `"extensions": {
        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
      }`

File: optimism.tokenlist.json [hunk 3]
```diff
@@ -1367,9 +1367,25 @@
...
+    },
+    {
+      "chainId": 42,
+      "address": "0xC99Fc8E8852e5BaB249174a983688c7F28065fB7",
+      "name": "KIDDO",
+      "symbol": "KIDDO",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/KIDDO.png"
```
Comment (Nicca42) [inline optimism.tokenlist.json:1378]: Additionally, you need to provide a bridge address here for your kovan token.

File: optimism.tokenlist.json [hunk 4]
```diff
@@ -1367,9 +1367,25 @@
...
+    {
+      "chainId": 69,
+      "address": "0x0E150963D01e8453B7099a3C032345527A4BAfFD",
+      "name": "KIDDO",
+      "symbol": "KIDDO",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/KIDDO.png",
       "extensions": {
-        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+        "optimismBridgeAddress": "0x0CFb46528a7002a7D8877a5F7a69b9AaF1A9058e"
```
Comment (Nicca42) [inline optimism.tokenlist.json:1388]: Just checking you are aware that you are using the BitBTC bridge here and NOT the standard bridge.

**Review by Nicca42 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**